_includes/head_custom.html

```diff
@@ -19,7 +19,7 @@
     });
 
     /* Read local storage state. */
-    if (localStorage.getItem('darkMode')) {
+    if (localStorage.getItem('darkMode') == 'true') {
```

Comment:
For completeness, can we make the `localStorage.setItem` call `String(setDark)` instead so that it’s clear it becomes a string?